ecosystem/sep-0038.md

```diff
@@ -172,6 +172,10 @@ Name | Type | Description
 
 This endpoint can be used to fetch the [indicative](https://www.investopedia.com/terms/i/indicativequote.asp) prices of available off-chain assets in exchange for a Stellar asset and vice versa.
 
+These prices are just indicatives provided at a given time, the actual price will be calculated according with the market price at conversion time, after the Anchor receives the funds from a User.
+
+It is expected that the prices returned **include a margin for the provider as a service fee**, and this margin will vary depending on the directional flow of funds.
```

Comment:
Can you remove the emphasis (bold)? SEPs contain a lot of important information and to bold one piece of information over the other implies there's areas of text that can be ignore.

Reply:
https://github.com/stellar/stellar-protocol/pull/987/commits/a727baf7a92e2e581a3fbd9c7295759b3a741605